sites/ol-author-similarity/index.js

```diff
@@ -56,9 +56,15 @@ function goBtnClicked(){
     .then(authorWorks => {
         const groups = findGroupsOfSimilar(authorWorks);
         document.getElementById("results").innerHTML = generateResultsHTML(groups, authorId);
+        document.getElementById("author-link").innerHTML = generateShowAuthorHTML(authorId);
     })
 }
 
+function generateShowAuthorHTML(authorId) {
+  let html = "<a href="https://openlibrary.org/authors/${authorId}">${authorId}</a>"
+  return html
+}
+
```

Comment:
```suggestion
function generateShowAuthorHTML(authorId) {
  return `<a href="https://openlibrary.org/authors/${authorId}">${authorId}</a>`;
}

```
In order to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) you need to replace " with `